docs/examples/CollapsableNav.js

```diff
@@ -0,0 +1,23 @@
+var navbarInstance = (
+    <Navbar brand="React-Bootstrap" toggleNavKey={0}>
+      <CollapsableNav eventKey={0}> {/* This is the eventKey referenced */}
+        <Nav navbar={true}>
+          <NavItem eventKey={1} href="#">Link</NavItem>
+          <NavItem eventKey={2} href="#">Link</NavItem>
+          <DropdownButton eventKey={3} title="Dropdown">
+            <MenuItem eventKey="1">Action</MenuItem>
+            <MenuItem eventKey="2">Another action</MenuItem>
+            <MenuItem eventKey="3">Something else here</MenuItem>
+            <MenuItem divider />
+            <MenuItem eventKey="4">Separated link</MenuItem>
+          </DropdownButton>
+        </Nav>
+        <Nav navbar={true} right={true}>
```

Comment:
When using booleans the mere presence of a props is presumed to be true. The following is also valid and cleaner, IMHO: `<Nav navbar right>`